enhancements/cloud-integration/infrastructure-external-platform-type.md

```diff
@@ -0,0 +1,586 @@
+Due to operators intended to react on the "External" platform the same as for the "None" one during the first phases,
+gating these API extensions does not seem needed.
+
+Related discussion: [1](https://github.com/openshift/enhancements/pull/1234#discussion_r968935259)
+
+---
+**Q**: Should we invest in preparing workflows that will perform UPI with the "None"/"External" platform types installation
+on the AWS or GCP, or existing vSphere-based workflows would be enough?
+
+**A**: Adding a job on one additional cloud platform to ensure that the "External" platform type works as intended looks reasonable now,
+but we should mainly rely on in-repo functional tests on a per-component basis and avoid engaging of e2e workflows as much as possible.
+
+Related discussions: [1](https://github.com/openshift/enhancements/pull/1234#discussion_r968934725), [2](https://github.com/openshift/enhancements/pull/1234#discussion_r968936500).
+
+---
+**Q**: Do we need an API for the MAPI components similar to the proposed [CCM one](#api-extensions)
+to allow users to choose how the MAPI components are deployed.
+
+**A**: We do not see an absolute necessity to add such API knobs right now. For the moment combination of the
+"External" platform and an enabled MAPI capability look like a sufficient signal.
+By the nature of the [capabilities mechanism](https://github.com/openshift/enhancements/blob/master/enhancements/installer/component-selection.md#summary)(i.e., no MAO deployment, no MAPI CRDs),
+if Machine API operator running and detects the "External" platform, that is the signal to it that *someone* is going to run a machine controller.
+
+Given that capabilities "API" is already added and in use, introducing additional knobs which will interfere with that
+would mean more code changes and the necessity of establishing a mechanism for communicating with users (i.e., what should handle such API if MAPI is disabled by the capabilities mechanism).
+
+However, if during upcoming phases, we will discover a need to add an API field specifically to help a user install their machine API controller,
+we could update this EP, or a new enhancement document should be created to provide a proper exploration of interaction with capabilities and other operation modes.
+
+Related discussions: [1](https://github.com/openshift/enhancements/pull/1234#issuecomment-1271896532)
+
+### Test Plan
+
+During the first stages, we must ensure that OpenShift [built-in operators whose behaviour depends on platform type](#affected-components) treat the "External" platform type the same way as "None".
+To achieve this - existing infrastructure and mechanisms employed for exercising topologies with the "None" platform type might be used.
+
+For the moment of writing, the only workflow where the "None" platform tests is [upi-conf-vsphere-platform-none-ovn](https://steps.ci.openshift.org/workflow/upi-vsphere-platform-none-ovn).
```

Comment:
@lobziik @elmiko if you would like to mention: as commented [here](https://github.com/openshift/enhancements/pull/1234#discussion_r1004433669), we added the vSphere with platform "None" installations to run the Provider Certification. We are considering of add-in AWS too, maybe with low frequency. As the Provider Certification and Platform "External" are strongly connected by the effort of the Installer Flexibility workgroup and Provider Certification program, I would suggest considering using that workflow too.
What do you think all (+ @dhellmann @bostrt @rvanderp3)?